torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -589,6 +590,16 @@ ProcessGroupNCCL::ProcessGroupNCCL(
             << "\nUSE_HIGH_PRIORITY_STREAM: "
             << options_->is_high_priority_stream
             << "\nNCCL_DEBUG: " << ncclDebugLevel;
+
+#ifdef USE_NCCL_WITH_UCC
+  static std::once_flag initialize_ucc_lib_flag;
+  std::call_once(initialize_ucc_lib_flag, [&]{
+    uccLib_ = loadTorchUCC();
+    if (uccLib_ != nullptr) {
+      LOG(INFO) << "[Rank " << rank_  << "] torch_ucc.so loaded";
+    }
```

Comment:
nit: In an else clause should we log that torch_ucc.so was not loaded.

Reply:
This is based on https://github.com/pytorch/pytorch/pull/69552#discussion_r765376633